crypto/evp/keymgmt_lib.c

```diff
@@ -102,10 +104,11 @@ void *evp_keymgmt_export_to_provider(EVP_PKEY *pk, EVP_KEYMGMT *keymgmt,
             return NULL;
 
         for (j = 0; j < i && pk->pkeys[j].keymgmt != NULL; j++) {
-            int (*exportfn)(void *provctx, OSSL_CALLBACK *cb, void *cbarg) =
+            int (*exportfn)(const EVP_KEYMGMT *keymgmt, void *provdata,
```

Comment:
A typedef is probably better.. especially if the function signature is going to change. (Again feel free to disagree)

Reply:
I would if it was going to get used in more places, but as far as I'm concerned, this is the only spot where using a function pointer for this particular signature, so...

(don't you wish that C had a `typeof` that you could give a function name, and that would be the signature of that function?  Alas...)